src/Controller/Adminhtml/Export/Feed.php

```diff
@@ -71,21 +71,40 @@ public function __construct(
     public function execute()
     {
         $result = $this->jsonResultFactory->create();
-
         try {
             preg_match('@/store/([0-9]+)/@', (string) $this->_redirect->getRefererUrl(), $match);
             $storeId = (int) ($match[1] ?? $this->storeManager->getDefaultStoreView()->getId());
-            $this->storeEmulation->runInStore($storeId, function () use ($storeId) {
+            $messages = [];
+
+            $this->storeEmulation->runInStore($storeId, function () use ($storeId, &$messages) {
+
                 $filename = (new FeedFileService())->getFeedExportFilename($this->feedType, $this->communicationConfig->getChannel());
                 $stream   = $this->csvFactory->create(['filename' => "factfinder/{$filename}"]);
                 $this->feedGeneratorFactory->create($this->feedType)->generate($stream);
-                $this->ftpUploader->upload($filename, $stream);
-                if ($this->communicationConfig->isPushImportEnabled($storeId)) {
-                    $this->pushImport->execute($storeId);
+                $filePath = $stream->getFeedFilePath();
```

Comment:
can we rename it to `getExportPath`? I know this would mean almost the same but it's different than `getFeedExportFilename` which is used few lines above